Makefile

```diff
@@ -6,6 +6,10 @@ help:
 build:
 	mvn clean install -DskipTests=true -Dgpg.skip=true -Dcheckstyle.skip=true -Dcheckstyle.skip=true -Ddependency-check.skip=true -Djavadoc.skip=true
 
+## build-deps - Build the project dependencies
+build-deps:
+	mvn install
```

Comment:
Question: Shouldn't this instead be in the `install` target instead of its own like all the other libs?

Reply:
When it's in "install", it's run as part of CI.
There (and locally), it throws an error due to the GPG plugin configuration.
Separating this out into a separate step (one that doesn't need to be run manually in most cases unless something goes wrong) is, IMO, a better backup.
For reference, this is the solution to the templating situation (explained in the wall of text in `src/main/java-templates/com/easypost/EasyPost.java`

Reply:
Resolved